tasks/test_syntax.js

```diff
@@ -119,13 +119,14 @@ function assertCircularDeps() {
         // as of v1.17.0 - 2016/09/08
         // see https://github.com/plotly/plotly.js/milestone/9
         // for more details
-        var MAX_ALLOWED_CIRCULAR_DEPS = 18;
+        var MAX_ALLOWED_CIRCULAR_DEPS = 17;
 
         if(circularDeps.length > MAX_ALLOWED_CIRCULAR_DEPS) {
+            console.log(circularDeps.join('\n'));
```

Comment:
Just to see where I had made circular refs when this test was failing. So now if you want to see them all just set `MAX_ALLOWED_CIRCULAR_DEPS = 0` temporarily and run this.